### ISSUE TITLE
Lock down the Azure/sdk-release-request repo

### ISSUE THREAD

**Issue by maririos** [Azure/azure-sdk-tools#8919]
We want to avoid people to create release request manually. Only the tooling should have permissions to this repo

**Comment by praveenkuttappan** [comment]
I assume this is only create issue permission.

**Comment by maririos** [comment]
Correct. Block it from manual issues to be create from anyone that is not the azure-sdk bot

**Comment by jrferry101** [comment]
@maririos is this related to blocking releases not using the release planner or can people use the azure-sdk bot with other methods?
